shared/editor/nodes/Attachment.tsx

```diff
@@ -38,6 +39,23 @@ export default class Attachment extends Node {
         size: {
           default: 0,
         },
+        preview: {
+          default: false,
+        },
+        width: {
+          default: 300,
+        },
+        height: {
+          default: 424,
+        },
+        type: {
+          default: null,
+          validate: "string|null",
+        },
```

Comment:
```suggestion
        contentType: {
          default: null,
          validate: "string|null",
        },
```

Lets just store the contentType for all future attachments, feels easier